index.src.html

```diff
@@ -519,8 +520,8 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   execute the following step:
 
   9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+      <a>`Clear-Site-Data`</a>, then execute <a>clear data for |response|</a>
+      algorithm on |response|.
```

Comment:
`<a>clear data for |response|</a>` seems fine: "algorithm on |response|" is redundant.